magento/magento2
# Increase DB_MAX_VARCHAR_LENGTH to 1024

Increase DB_MAX_VARCHAR_LENGTH to 1024

### Description (*)
Increase the length to fix the issue described here https://github.com/magento/magento2/issues/15679

### Fixed Issues (if relevant)

1. Fixes magento/magento2#15679

### Manual testing scenarios (*)
Log into backend panel
Go to System->Import
Select products as Entity Type option, select Add/Update as Import Behavior, Select Stop on Error, Allowed Errors Count is 10,Field separator is Comma(,) ,Multiple value separator is Comma(,) , Fields enclosure filed is empty.
4.Select File to Import
5.Images File Directory left blank (As the images were going to fetch from Amazon.com)
6.Click Check Data

### Contribution checklist (*)
